[FM v0.15.0]--support YAML config for java runtime, support log format json for java and python (#273)
streamnative/function-mesh-website: docs/reference/crd-config/function-crd.md

| `pyLocation` | The path to the JAR file for the function. It is only available for Pulsar functions written in Python.|
| `extraDependenciesDir` | It specifies the dependent directory for the JAR package. |

## Runtime logs

> Runtime logs include all logs generated by your pods. These logs provide information about the output for your functions. Function Mesh provides the following fields for the runtime logs in the CRD definition.

Reviewer: ```suggestion
Runtime logs include all logs generated by your pods. These logs provide information about the output of your functions. Function Mesh provides the following fields for the runtime logs in the CRD definition.
```

| Field | Description |
| --- | --- |
| `level` | The log level. Available options are `off`, `trace`, `debug`, `warn`, `error`, `fatal`, `all`, and `panic`. For details about each log level, see [log levels](#log-levels).
| `rotatePolicy`| The log rotation policy. Available options are `TimedPolicyWithDaily`, `TimedPolicyWithWeekly`, `TimedPolicyWithMonthly`, `SizedPolicyWith10MB`, and `SizedPolicyWith100MB`. For details, see [log rotation policies](#log-rotation-policies).
| `format` | The log format that defines how the content of a log file should be interpreted. Available options are `json` and `text`. The log format configurations are only available for the Java and Python runtimes. |

> | `logConfig`| A key-name format option that is used to reference to a custom log configuration file. |

Reviewer: ```suggestion
| `logConfig`| A key-name format option used to reference a custom log configuration file. |
```

> | `javaLog4JConfigFileType` | The Log4j configuration file type. Available options are `yaml`, `xml` and `init`. This option is only available for the Java runtime. |

Reviewer 2: ```suggestion
| `javaLog4JConfigFileType` | The Log4j configuration file type. Available options are `yaml` and `xml`, and `xml` is the default value. This option is only available for the Java runtime. |
```
